frontend/src/app/main/ui/ds/product/milestone.scss

```diff
@@ -0,0 +1,164 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@use "../_sizes.scss" as *;
+@use "../_borders.scss" as *;
+
+// .empty-placeholder {
+//   display: grid;
+//   grid-template-columns: auto 1fr auto;
+//   place-content: center;
+//   background: none;
+//   color: var(--color-foreground-secondary);
+//   height: $sz-160;
+//   max-width: $sz-964;
+//   border-radius: $br-8;
+//   border: $b-1 solid var(--color-background-quaternary);
+// }
+//
+// .text-wrapper {
+//   display: grid;
+//   grid-auto-rows: auto;
+//   align-self: center;
+//   justify-self: center;
+//   max-width: $sz-400;
+// }
+//
+// .placeholder-title {
+//   color: var(--color-foreground-primary);
+// }
+//
+// .svg-decor {
+//   height: $sz-160;
+//   width: $sz-200;
+//   color: var(--color-background-quaternary);
+// }
+
+.milestone {
+  border: $b-1 solid var(--border-color, transparent);
+  border-radius: $br-8;
+
+  cursor: pointer;
+  background: var(--color-background-primary);
+
+  display: grid;
+  grid-template-areas:
+    "avatar name button"
+    "avatar content button";
+  grid-template-rows: auto 1fr;
+  grid-template-columns: calc(var(--sp-xxl) + var(--sp-l)) 1fr auto;
+
+  padding: var(--sp-s) 0;
+  align-items: center;
+
+  column-gap: var(--sp-s);
+
+  &.is-selected,
+  &:hover {
+    --border-color: var(--color-accent-primary);
+  }
+
+  .name-input {
+    --input-bg-color: none;
+    --input-outline-color: none;
+    grid-area: name;
+    padding: 0;
+    border: none;
+
+    input {
+      height: auto;
+      margin: 0;
+      font-family: "worksans", "vazirmatn", sans-serif;
+      font-optical-sizing: auto;
+      font-weight: 400;
+      line-height: 1.3;
+      font-size: 0.875rem;
```

Comment:
If this typography does not exist, we need to ask design to define it + add it to the Design System . Then, we should add this to the typography mixins and use its given name.